release-notes-agent/src/main/java/com/epam/release/notes/agent/service/action/ActionNotificationService.java

```diff
@@ -22,6 +22,7 @@
 
 public interface ActionNotificationService {
 
-    void process(String oldVersion, String newVersion, List<JiraIssue> jiraIssues, List<GitHubIssue> gitHubIssues);
+    void process(String oldVersion, String newVersion, List<JiraIssue> jiraIssues, List<GitHubIssue> gitHubIssues,
+                 String[] recipients);
```

Comment:
why `recipients` is an array? it would be convenient to have it as list, however since in the future we will have other actions I would prefer save `process` signature without `recipients` since it is very specific and should be encapsulated in specific ActionService